File: Pattern29.c
// a
// B c
// D e F
// g H i J
// k L m N o

#include <stdio.h>

void main()
{
    int n;
    printf("Enter the number:");
    scanf("%d", &n);
    int val = 97;
    int flag = 0;
    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= i; j++)
        {
            if (flag == 0)
            {
                printf("%c ", (char)(val));
                flag = 1;
            }
            else
            {
                val = val - 32;
                printf("%c ", (char)(val));
                flag = 0;
                val+=32;
            }
            val += 1;
        }
        printf("\n");
    }
}

File: Pattern29.py
n = int(input("Enter the number:"))
val = 97
flag = 0
for i in range(1,n+1):
    for j in range(1,i+1):
       if flag == 0:
            print(chr(val)," ",end='')
            flag = 1
       else:
            val = val - 32;
            print(chr(val)," ",end='')
            flag = 0
            val+=32
       val = val + 1
    print()